src/Catalyst.Cli.Tests/UnitTests/Commands/Response/GetPeerReputationResponseTests.cs

```diff
@@ -45,7 +45,7 @@ public void GetPeerReputationResponse_Can_Get_Output()
             //Arrange
             var getPeerReputationResponse = new GetPeerReputationResponse {Reputation = 100};
             var commandContext = TestCommandHelpers.GenerateCliResponseCommandContext(_testScheduler);
-            var getPeerReputationCommand = new PeerReputationCommand(commandContext, Substitute.For<ILogger>());
+            new PeerReputationCommand(commandContext, Substitute.For<ILogger>());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Either remove this useless object instantiation of class 'PeerReputationCommand' or use it.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4832783)